src/com/facebook/buck/go/GoTest.java

```diff
@@ -37,6 +37,7 @@
 import com.facebook.buck.test.TestRunningOptions;
 import com.facebook.buck.test.result.type.ResultType;
 import com.google.common.base.Charsets;
+import com.google.common.base.Function;
```

Comment:
I should have looked at the travis build.  This is an unused import, so `ant lint` fails.